index.d.ts

```diff
@@ -0,0 +1,161 @@
+/// <reference types="node"/>
+import {EventEmitter} from 'events';
+
+interface QueueAddOptions {
```

Comment:
All the types/interfaces that can be useful for the dev must be exported. Otherwise there's no way for the dev to define the options before passing them in, right?

Reply:
Yet another oversight.

Fun fact: this will be exported regardless of whether I write the `export` statement or not. You actually have to work quite hard if you don't want to export some interface from an ambient declaration.